db/seeds.rb

```diff
@@ -31,6 +31,18 @@ def update_or_create(model, attributes)
 }
 update_or_create(SourceType, :name => "amazon", :product_name => "Amazon Web Services", :vendor => "Amazon", :schema => amazon_json_schema)
 
+azure_json_schema = {
+  :title  => "Configure Azure",
+  :fields => [
+    {:component => "text-field", :name => "role", :type => "hidden", :initialValue => "azure"},
+    {:component => "text-field", :name => "authtype", :type => "hidden", :initialValue => "access_key_secret_key"},
+    {:component => "text-field", :name => "extra.azure.tenant_id", :label => "Tenant ID"},
```

Comment:
@martinpovolny  @rvsia is this the right format for the component? I want this value to be sent as `:extra => {:azure => {:tenant_id => "tenant_id_value"}}.to_json`

like you can see in the spec here https://github.com/ManageIQ/sources-api/pull/62/files#diff-14ea593593f9669c3585d47cacefe071R58

Reply:
@martinpovolny can you check that these params are correct?